### DISCUSSION TITLE
Why the public contact api cannot create a phone_number variable?

### DISCUSSION THREAD

**Discussion by ramphy** [chatwoot/chatwoot#4013] [Chatwoot APIs] [answered]
Hello,

Why the public contact api cannot create a phone_number variable?

I'm  trying to use the public contact api, to integrate my API with Chatwoot.

https://app.chatwoot.com/public/api/v1/inboxes/{inbox_identifier}/contacts
{
  'email' : 'email2225@spam4.me',
  'phone_number' : '+584120695353',
  'avatar_url' : '',
}

I'm triying to put phone_number in the body request and is not saving the information.

Thank you.

**Comment by sojan-official** [comment, marked as answer]
@ramphy  I have logged https://github.com/chatwoot/chatwoot/issues/4023 
we will take this up for upcoming releases.

**Reply by ramphy** [reply]
Thank you for your answer!
